src/Spago/CLI.hs

```diff
@@ -157,6 +161,8 @@ parser = do
     nodeArgs         = many $ CLI.opt (Just . BackendArg) "node-args" 'a' "Argument to pass to node (run/test only)"
     backendArgs      = many $ CLI.opt (Just . BackendArg) "exec-args" 'b' "Argument to pass to the backend (run/test only)"
     replPackageNames = many $ CLI.opt (Just . PackageName) "dependency" 'D' "Package name to add to the REPL as dependency"
+    scriptPackageNames = many $ CLI.opt (Just . PackageName) "dependency" 'd' "Package name to add to the script as a dependency"
+    scriptSource = CLI.arg Just "source" "Source file to run as script"
```

Comment:
In the example given in #683 the command used `-d` to indicate a dependency. However, the `repl` command above uses `-D` for dependencies. It seems better to be consistent and use the same for both commands -- it's `repl` that seems to be the odd one out, though, as the other commands (like `nodeArgs` and `backendArgs`) use lowercase characters.

Reply:
Yes I think consistency between the two commands is very important. I'm fine changing the `repl` to use lowercase `d`, if there are no other flags using that letter

Reply:
Hmm, it looks like lowercase `d` is taken by `deps-only`. 

Since that is the case, maybe we should make both `repl` and `script` use uppercase `D`?

Reply:
@colinwahl sounds great!

Reply:
Thinking more about this, I think it would be nicer to use the `d` letter for passing in dependencies - I think it would be fine to deprecate the shorthand for `deps-only` - allowing only the long form there

Reply:
Sure.